lib/harvesting/models/user.rb

```diff
@@ -26,11 +27,13 @@ class User < HarvestRecord
                  :created_at,
                  :updated_at
 
-      private
-
       def path
         @attributes['id'].nil? ? "users" : "users/#{@attributes['id']}"
       end
+
+      def name
+        @attributes['name'].nil? ? "#{first_name} #{last_name}" : @attributes['name']
```

Comment:
@mscottford Is this something that exists but is undocumented in the Harvest API? 

I don't see this attribute documented in their API: https://help.getharvest.com/api-v2/users-api/users/users/

Just curious...

Reply:
That attribute shows up in some places where user is listed in nested JSON. Take a peek at the example output for getting all user assignments. https://help.getharvest.com/api-v2/projects-api/projects/user-assignments/

Reply:
@mscottford Interesting! 👍